app/models/tag.py

```diff
@@ -11,8 +11,6 @@ class TagManager(models.Manager):
     def create_tag(self, initiator: Player, receiver: Player, tagged_at: datetime, location: str, description: str,
                    point_modifier: int = 0) -> 'Tag':
 
-        if initiator.is_spectator or receiver.is_spectator:
```

Comment:
don't we need additional checks?

Reply:
I'm not sure, considering that having the `player_required` decorator on the tag report/supply code views means you can't even access that content afaik?